_posts/2022-01-xx-renaissance-0-14-0.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: mainpost
+projectname: Renaissance Suite
+title:  "Renaissance 0.14 Released"
+author: Lubomír Bulej
+---
+
+We are pleased to announce a new release of the Renaissance benchmark suite.
+This release contains a mix of benchmark dependency updates and some internal
+changes. The most visible change is an update of Apache Spark to version 3.2.0,
+which enabled migrating the `apache-spark` benchmarks from Scala 2.12 to Scala
+2.13. In addition, Scala 2.13 benchmarks were updated to use Scala 2.13.8. The
+internal changes make it easier to run individual benchmarks in what we call
+_standalone mode_, which is desired in some use cases.
```

Comment:
"which is desired for GraalVM Native Image support"

Reply:
Done.